server/models/vote.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const VoteSchema = new Schema({
+  userId: {
+    type: Schema.Types.ObjectId,
+    required: true,
+    ref: 'users',
```

Comment:
we definitely want to add an index to this field since we are going to be fetching this so often 
https://mongoosejs.com/docs/2.7.x/docs/indexes.html

Reply:
Would you say that it's a good idea to also add an index on `pollId` since they're both fetched together? This is how I fetch them for `DELETE /polls/:pollId/:imageIdx/vote`:
```JavaScript
Vote.find({
  $and: [{ userId: req.user._id }, { pollId: req.params.pollId }],
});
```

Reply:
I used a compound index (ref: 75370eb) on both `userId` and `pollId` since they'll be always fetched together for `DELETE.

What do you think?

Reply:
Yes that makes sense... if you only fetch in compound, then you don't need the individual indices